### DISCUSSION TITLE
Can the vvenc support input yuv loop encoding?

### DISCUSSION THREAD

**Discussion by MichaelMM12** [fraunhoferhhi/vvenc#307] [Q&A] [answered]
Hello All,
I want to encode a long frame vvc video, and want to repeat the same yuv for the encoding, but I found no parameter can support this.
I set the FramesToBeEncoded to a  very big value, but the output only show the same frame number with yuv.
Thanks!

**Comment by lehmann-c** [comment, marked as answer]
That is not possible with VVenC, but you can use third-party apps as ffmpeg to loop the input and pipe it into the vvenc application.
e.g. ffmpeg with the options `-stream_loop`
`ffmpeg -stream_loop 1 -s 1920x1080 -pix_fmt yuv420p10le -i myinput1080p.yuv  -vcodec rawvideo -f rawvideo - | vvencapp -i - -s 1920x1080 -c yuv420_10  -o out.266`

That will loop the 10bit RAW file once..

**Reply by MichaelMM12** [reply]
thanks for the help! let me try ffmpeg.
